homeassistant/components/denonavr/media_player.py

```diff
@@ -158,9 +159,19 @@ def unique_id(self):
     @property
     def device_info(self):
         """Return the device info of the receiver."""
-        return {
-            "identifiers": {(DOMAIN, self._receiver_device_id)},
+        device_info = {
+            "config_entry_id": self._config_entry.entry_id,
```

Comment:
We don't need to set this. It's set by the core.

Reply:
I have removed it